### DISCUSSION TITLE
Placeholder and Omitted Features Tracker

### DISCUSSION THREAD

**Discussion by N7Huntsman** [CombatExtended-Continued/CombatExtended#795] [General]
This post will serve as a centralized list of features in compatible mods that have been removed or only partially implemented so that they can be tracked and resolved--please include a link to the associated mod. Ideally, in the these features will be made fully compatible.

**Comment by N7Huntsman** [comment]
**Vanilla Furniture Expanded - Security**
~~Currently, the Artillery turret's inter-map bombardment feature is incompatible and has been disabled.~~
~~https://steamcommunity.com/sharedfiles/filedetails/?id=1845154007~~

Resolved w/ c# patch.

**Comment by N7Huntsman** [comment]
**Vanilla Factions Expanded - Mechanoids**
The Knight flamethrower mechs use a CE-style flamethrower, rather than the Vanilla Expanded kind with its custom graphics. Additionally, there is still no way for player-controlled mechs to handle ammo. For the time being, the combat robot's weapon has been given infinite ammo.
https://steamcommunity.com/sharedfiles/filedetails/?id=2329011599

**Comment by ghost** [comment]
**Vanilla Weapons Expanded**
Muskets generate a cloud of smoke after firing, had to be removed due to incompatibilities.

**Vanilla Weapons Expanded - Heavy**
Flamethrower uses a VE style flamethrower with custom graphics, this was replaced with a CE style flamethrower.

**Comment by N7Huntsman** [comment]
**Alpha Animals**
The Mature Fleshbeast's "swallowed whole" attack/mechanic has been disabled, as its interaction with CE's predator attacks against downed pawns means it generally insta-kills its victim, rather than swallowing them alive. 

**Rimmu-Nation Security**
The claymore mine has been removed, as the penetration of the shrapnel does not use CE armor penetration, making the mine very weak.

**Comment by Breadbox2k19** [comment]
> **Alpha Animals**
> The Mature Fleshbeast's "swallowed whole" attack/mechanic has been disabled, as its interaction with CE's predator attacks against downed pawns means it generally insta-kills its victim, rather than swallowing them alive.
> 
> **Rimmu-Nation Security**
> The claymore mine has been removed, as the penetration of the shrapnel does not use CE armor penetration, making the mine very weak.

Can't the Claymore be patched to use better fragments?

**Comment by N7Huntsman** [comment]
> Can't the Claymore be patched to use better fragments

Tried that, didn't work. Seems to have less to do with the projectiles themselves and more to do with them not having a proper source.
